torch/csrc/inductor/aoti_torch/c/shim_deprecated.h

```diff
@@ -0,0 +1,151 @@
+#ifndef AOTI_TORCH_SHIM_DEPRECATED
+#define AOTI_TORCH_SHIM_DEPRECATED
+
+#include <stddef.h>
+#include <stdint.h>
+
+// WARNING: This header file contains to-be-deprecated APIs for aten ops that
+// will be removed in the future. Now shims for aten ops are generated by
+// torchgen and locate in torch/csrc/inductor/aoti_torch/generated
+
+#ifdef __GNUC__
+#define AOTI_TORCH_EXPORT __attribute__((__visibility__("default")))
```

Comment:
This and the following macros will cause duplicated macro definition, which is not recommended. You can split them into a separate macros. h header file.